main.go

```diff
@@ -148,6 +148,12 @@ func Run(client *cmd.Client, args ...string) {
 			Usage:  "Removes a specific bridge",
 			Action: client.RemoveBridge,
 		},
+		{
+			Name:    "agree",
+			Aliases: []string{"csa", "creatsa"},
```

Comment:
`creatsa` or `createsa`?

optional nitpick: I would remove `csa` because I worry too many aliases will make the CLI look cluttered, especially in the help prompt.